dev/set_matrix.py

```diff
@@ -658,7 +658,6 @@ def main(args):
     print(divider("Parameters"))
     print(json.dumps(args, indent=2))
     matrix = generate_matrix(args)
-    return
```

Comment:
Once https://github.com/astral-sh/ruff/pull/10891 lands, we can enable this rule to detect dead code.

Reply:
Nice, then I'll wait for that PR. It started a while ago but seems still quite active.